src/odemis/gui/cont/tabs/cryo_chamber_tab.py

```diff
@@ -910,6 +924,37 @@ def _sample_heater_to_va(self):
     def Show(self, show=True):
         Tab.Show(self, show=show)
 
+        # the first time the tab is shown, ask the user to create or load a project
+        if self.IsShown() and not self._is_initial_project_ready:
+
+            # stay in the loop until the user successfully creates or loads a project
+            while True:
```

Comment:
Just wondering if you can do away with this while statement. If not, its fine!

Reply:
No, we want the user to stay here until they either create a new project or load an existing one. So if they fail to do so, they get put back to the start of the flow until they do so correctly.